ingestion/src/metadata/ingestion/source/database/postgres/metadata.py

```diff
@@ -73,23 +86,8 @@ class POLYGON(String):
     __visit_name__ = "POLYGON"
 
 
-@reflection.cache
-def get_table_names(
-    self, connection, schema=None, **kw
-):  # pylint: disable=unused-argument
-    """
-    Overwriting get_table_names method of dialect to filter partitioned tables
-    """
-    result = connection.execute(
-        sql.text(POSTGRES_GET_TABLE_NAMES).columns(relname=sqltypes.Unicode),
-        dict(schema=schema if schema is not None else self.default_schema_name),
-    )
-    return [name for name, in result]
-
-
 ischema_names.update({"geometry": GEOMETRY, "point": POINT, "polygon": POLYGON})
 
-PGDialect.get_table_names = get_table_names
```

Comment:
we don't need to monkey-patch the dialect here, just run our query inside `query_table_names_and_types` and get the required columns